.github/workflows/runTests.yml

```diff
@@ -16,4 +16,4 @@ jobs:
         run: npm ci
 
       - name: Run all vitest tests
-        run: npm run test
+        run: npm run aegtest
```

Comment:
Since you use npx for the playwright tests, it would be more clear to use npx vitest here and get rid of the aegtest script.